### DISCUSSION TITLE
Adozione dall'estero: indicazioni caso d'uso se non attiva Convenzione Aja 29.5.1993

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1039] [Supporto servizi cooperativi]
Qual è il caso d'uso di riferimento per la _trascrizione della sentenza del Tribunale per i Minorenni che dichiari efficace il provvedimento straniero di adozione_ laddove in cui il paese straniero in questione non aderisca alla Convenzione Aja 29.5.1993

**Comment by acolosimosogei** [comment]
Salve, stiamo lavorando alla discussion #1039, le daremo un riscontro quanto prima.
Grazie.

**Comment by asqueglia** [comment]
Buongiorno la segnalazione è stata corretta con la versione 1.35.0 aggiungendo il caso
